com.chartboost.mediation/Runtime/Plugins/iOS/HeliumSdkManager.mm

```diff
@@ -97,10 +111,7 @@ const void serializePlacementLoadWithError(NSString *placementName, NSString *re
     if (price == nil)
         price = 0;
     
-    if (error != nil)
-        errorMessage = error.description.UTF8String;
-    
-    placementLoadEvent(placementName.UTF8String, requestIdentifier.UTF8String, auctionId.UTF8String, partnerId.UTF8String, [price doubleValue], errorMessage);
+    placementLoadEvent(placementName.UTF8String, requestIdentifier.UTF8String, auctionId.UTF8String, partnerId.UTF8String, [price doubleValue], formatError(error));
```

Comment:
`nit:` Is there a way that we can make it so that we don't have to include UTF8String and have it be done beforehand?